### PR TITLE
Add supervision controller to get results count

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ */
+@Service
+public class SupervisionService {
+    private final GlobalStatusRepository globalStatusRepository;
+
+    public SupervisionService(GlobalStatusRepository globalStatusRepository) {
+        this.globalStatusRepository = globalStatusRepository;
+    }
+
+    public Integer getResultsCount() {
```
Comment (sBouzols) [inline src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java:23]: Why not getting `resultsCount` from the ResultRepository ?
Maybe even both could be useful to detect a missmatch with globalStatus count maybe ?

**Comment by SlimaneAmar** [comment]
Remove code smell

File: src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java:23]: Not after 
[https://dev.azure.com/gridsuite/MVP/_workitems/edit/2441](https://dev.azure.com/gridsuite/MVP/_workitems/edit/2441)

**Review by SlimaneAmar — APPROVED** [review]

File: src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java (hunk 1 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/shortcircuit/server/service/SupervisionService.java:23]: Done

**Review by sBouzols — APPROVED** [review]

**Pull request merged**